tools/accuracy/accuracy_checker.py

```diff
@@ -88,6 +88,12 @@ def parse_args():
                         default=False,
                         help='Turn on ort VERBOSE logging via session options')
 
+    parser.add_argument('--show_test_data',
```

Comment:
Use dashes not underscores for command line arguments.

Reply:
Done. Thanks!